src/components/styles.css

```diff
@@ -0,0 +1,80 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+}
+
+body {
+  background-color: rgb(233, 224, 224);
+}
```

Comment:
- [ ] Your project has been styled incredibly well :+1:, however, there are a few color-related problems that need to be addressed. Specifically, the background color should be #fafafa, the navbar should be #fff, the title should be #0290ff, and so on. Please ensure that the colors of your elements match the [template](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) provided by Microverse, so that the project maintains a consistent and professional appearance.
- To find the design, [login to Zeplin](https://app.zeplin.io/login) using this credentials
  - username: `student@microverse.org`
  - password: `jb077mL2dL0Dfx`

**Your design**

![Screenshot from 2023-04-13 19-09-24](https://user-images.githubusercontent.com/105117832/231909829-362c1e5d-dd24-42e0-836c-4a7b1ad44d96.png)

**Template**

![Screenshot from 2023-04-13 19-15-56](https://user-images.githubusercontent.com/105117832/231909873-f6cea4d7-5026-4906-8806-1c66d751a60b.png)